_posts/2019-08-23-gsoc_LibrecoresCI_Report.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: post
+title: "GSoC Project Report: Continuous Integration for Hardware Projects on LibreCores CI"
+description: ""
+category:
+tags: ['GSoC', 'LibrecoresCI' ,'FOSSi-Foundation']
+author: Nancy Chauhan
+---
+
+Librecores provides a platform to share projects and ideas, in the area of free and open-source digital hardware design. Librecores CI is a service to provide CI for hardware projects hosted on Librecores to improve user experience and reliability.
+
+In these summers I worked on developing continuous integration solution for hardware Projects such as OpenRISC, a family of free and open-source processor implementations on the RISC architecture.
```

Comment:
Grammar:
**This summer** I worked on developing continuous integration **pipeline** for hardware **projects** such as OpenRISC, a family of free and open-source processor implementations on the RISC architecture.